packages/components/src/InputTime/hooks/useTimePredict.ts

```diff
@@ -1,6 +1,6 @@
 import debounce from "lodash/debounce";
 import { useCallback, useEffect, useState } from "react";
-import { InputTimeProps } from "../InputTimeProps";
```

Comment:
minor rename to better match how we name these types of files in the project

Reply:
Minor comment, most of our types files are ___.**t**ypes.ts not ___.**T**ypes.ts